source/jobs/pause-resume-kafka-jobs.txt

```diff
@@ -0,0 +1,77 @@
+.. _rm-pause-resume-kafka-jobs:
+
+================================
+Pause and Resume Kafka Sync Jobs
+================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can run a sync job with Kafka support, when your sync job is using 
+Kafka you have the ability to pause and resume. The amount of 
+time that a job can be paused depends on the log retention period of
+your source relational database.
+
+About this Task
+---------------
+
+- This task outlines the pause and resume process for **Kafka** or 
+  **Confluent Cloud Kafka** supported sync jobs only. If you are running 
+  Relational Migrator without Kafka support you are not able to pause or 
+  resume sync jobs.
+
+.. include:: /includes/fact-kafka-additional-time.rst
+
+Before you Begin
+----------------
+
+Enable Kafka support for relational migrator, for details see the Kafka
+or Confluent Cloud Kafka install docs. **Todo** add links to the appropriate 
+installation reference pages.
+
+Steps
+-----
+
+.. procedure::
+   :style:  connected
+
+   .. step:: Create a continous sync job
+     
+      Create a continuous sync job, for details see the 
+      :ref:`create a sync job page <rm-create-jobs>`.
+
+   .. step:: Pause the sync job
+     
+      a. Wait for the sync job to enter the CDC stage. Once the sync job 
+         enters the CDC stage click the :icon-fa5:`pause` :guilabel:`Pause` 
+         button on the :guilabel:`Job Overview` 
+         pane. 
+
+      #. Click :guilabel:`Pause CDC` on the pop-up modal to confirm
+         you would like to pause the job. The status of the 
+         :guilabel:`CDC Stage` updates to :guilabel:`Paused`.
+
+         .. warning::
+
+            You must resume the sync job within your source 
+            database's CDC retention period to ensure no events are lost.
```

Comment:
Not sure if we want to add a reference SQL query here for the various databases to check what the current retention period is. If the Relational Migrator team happens to have a query handy I can add a tabbed code block so that users can check and there are no surprises.